adsingestp/parsers/jats.py

```diff
@@ -889,9 +889,10 @@ def _parse_page(self):
         if self.article_meta.find("counts") and self.article_meta.find("counts").find(
             "page-count"
         ):
-            self.base_metadata["numpages"] = (
-                self.article_meta.find("counts").find("page-count").get("count", "")
-            )
+            if self.article_meta.find("counts").find("page-count").get("count", "").isdigit():
```

Comment:
The `isdigit()` will throw an exception if count is anything other than a string -- I'm not 100% certain beautifulsoup will always render "count" as a string if it's in the file as `<page-count count=3/>` rather than `<page-count count="3"/>`.